packages/core/admin/ee/server/src/audit-logs/services/lifecycles.ts

```diff
@@ -105,6 +95,7 @@ const createAuditLogsService = (strapi: Core.Strapi) => {
     }
 
     // Ignore some events based on payload
+    // TODO: What does this ignore in upload? Why would we want to ignore anything?
     const ignoredUids = ['plugin::upload.file', 'plugin::upload.folder'];
     if (ignoredUids.includes(args[0]?.uid)) {
```

Comment:
@Feranchz do you have any idea what this about? As far as I can tell there isn't anything to ignore from the upload plugin 🤔.